var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -261,7 +262,7 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     # https://github.com/protocolbuffers/protobuf/issues/10051
     # https://github.com/tensorflow/tensorflow/issues/56266
     depends_on("py-protobuf@:3.19", type=("build", "run"), when="@:2.11")
-    depends_on("py-protobuf+cpp", type=("build", "run"))
```

Comment:
@manuelakuhn going to try to relax the dependency you noted in #38614 now that this feature is no longer available in newer releases of protobuf.